genie-web/src/test/java/com/netflix/genie/web/controllers/RestControllerIntegrationTestsBase.java

```diff
@@ -115,6 +117,21 @@
         .setDateFormat(new GenieDateFormat())
         .registerModule(new Jdk8Module());
 
+    protected final Matcher<String> dateFormatMatcher = new BaseMatcher<String>() {
```

Comment:
This part should probably be ported into `master` after this is merged

Reply:
Yes, i left the controller tests into a separate commit so we can cherry-pick